src/components/chat/chatBanner/ChatBanner.tsx

```diff
@@ -0,0 +1,27 @@
+// import Image from "../../../Images/logo192.png";
+import Typography from "@material-ui/core/Typography";
+import { green } from "@material-ui/core/colors";
+import '../../../assets/stylesheets/chat/chatBanner.scss'
+
+const ChatBanner = () => {
+  return (
+    <>
+      <div className="banner">
+        <div className="bannerAvatar">
+          <img alt="Chat Avatar" src='https://cdn4.iconfinder.com/data/icons/logos-3/600/React.js_logo-1024.png' width="20" />
```

Comment:
we can import existing logo from assets/images instead of hardcoding the URL.

Reply:
nice catch! addressed https://github.com/Simhanischal/Mini-ChatGPT/commit/c4184336518cc40c5e2c1869abff852d53611f2d